### DISCUSSION TITLE
Stream Deck store plugin icon

### DISCUSSION THREAD

**Discussion by nhan-rao** [elgatosf/elgato#44] [Help]
I'm getting ready to submit a plugin to the Stream Deck store, but I can't seem to figure out how to designate the icon used in the store when users go to download it. I don't see information on it in the style guide or the distribution instructions. It doesn't seem to be the category or icon, since those are supposed to be one color but the store icons are full color. Does anyone know?

**Comment by ChekTek** [comment]
Hi!

The plugin used in the store can be full color. You can set the plugin icon using the "Icon" field in the manifest.json.

https://docs.elgato.com/sdk/plugins/manifest#members

```
The relative path to a png image without the .png extension. This image is displayed in the Stream Deck marketplace. The PNG image should be a 288px by 288px. You should provide @1x and @2x. The Stream Deck application takes care of loading the appropriate version of the image.
```
